projects/igniteui-angular/src/lib/grids/grid/grid-toolbar.spec.ts

```diff
@@ -84,47 +95,47 @@ describe('IgxGrid - Grid Toolbar #grid', () => {
 
     it('testing main export button visibility', () => {
         grid.showToolbar = true;
-        grid.toolbarTitle = 'Grid Toobar Title';
+        grid.toolbarTitle = TOOLBAR_TITLE;
         fixture.detectChanges();
 
         const gridToolbar = getToolbar(fixture);
 
-        let exportButton = gridToolbar.query(By.css('.igx-grid-toolbar__dropdown#btnExport'));
+        let exportButton = gridToolbar.query(By.css(CSS_CLASS_EXPORT_BTN));
```

Comment:
You can use GridFunctions.getExportButton(fix)
You can also update the methods in the GridFunctions for tollbar to use constants for clases